src/js/tracks/video-track-enums.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @file video-track-enums.js
+ */
+
+/**
+ * https://html.spec.whatwg.org/multipage/embedded-content.html#dom-videotrack-kind
+ *
+ * enum VideoTrackKind {
+ *   "alternative",
+ *   "captions",
+ *   "main",
+ *   "sign",
+ *   "subtitles",
+ *   "commentary",
+ *   "",
+ * };
+ */
+const VideoTrackKind = {
```

Comment:
@gkatsev the spec does not define an actual enumeration type for `kind`, there's just [a table of strings](https://html.spec.whatwg.org/multipage/embedded-content.html#dom-TrackList-getKind-categories). Contrast that with the [network or ready states](https://html.spec.whatwg.org/multipage/embedded-content.html#htmlmediaelement) on the media element, which do have runtime constants.

I don't understand your point about having a default `kind`. Would you mind elaborating?